modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -395,6 +397,34 @@ baseTest(
 	}
 );
 
+friendlyURLDuplicatedMessageTest(
```

Comment:
no need to create another test name. Just `baseTest('Check alert...') ` is fine.